core/node/node_sync/src/fetcher.rs

```diff
@@ -174,7 +174,8 @@ impl IoCursorExt for IoCursor {
                         block.l1_gas_price,
                     ),
                     first_l2_block: L2BlockParams {
-                        timestamp: block.timestamp,
+                        // It's ok that we lose info about millis since it's only used for sealing criteria.
+                        timestamp_ms: block.timestamp * 1000,
```

Comment:
- I find it weird that fetched block info loses information about the timestamp *per se*. Are we 100% sure we won't need it?
- Just to be sure: Do I understand correctly that the VM is still fed the timestamp in seconds? So it may have several sequential blocks with the same (seconds) timestamp?

Reply:
- Currently milliseconds are only used for sealing criteria and I'm sure that we don't need them in fetched block for this purpose. One potential use-case for millis would be to return them in API. Overall, I think it's pointless, we stick to compatibility with ethereum that uses seconds in VM and block header, so even if we persist millis to DB and include them in fetched blocks they won't be validated by protocol nor consensus.
- yes